spec/jobs/send_job_application_data_expiry_notification_job_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe SendJobApplicationDataExpiryNotificationJob do
+  let(:notification) { instance_double(Publishers::JobApplicationDataExpiryNotification) }
+  let(:organisation) { create(:school) }
+  let(:publisher) { create(:publisher, organisation_publishers_attributes: [{ organisation: organisation }]) }
+  let!(:vacancy_with_two_weeks_to_data_expiry) { create(:vacancy, expires_at: 351.days.ago, publisher: publisher, organisation_vacancies_attributes: [{ organisation: organisation }]) }
+  let!(:vacancy) { create(:vacancy, expires_at: 1.day.ago, publisher: publisher, organisation_vacancies_attributes: [{ organisation: organisation }]) }
+
+  before do
```

Comment:
very minor thing, when is one liner I prefer it like this:

`before { allow(DisableExpensiveJobs).to receive(:enabled?).and_return(false) }`